Svenbjorn/whatson
# fix/map-marker-placement

Monday: https://visitfaroeislands.monday.com/boards/4732496989/pulses/4533936046

Selected map icon should be in front of map.

## How to test
 - Go to a whatson map view
 - Select a map icon that overlaps other icons
 - Ensure that the selected icon never goes behind another icon.
